Ollivanders/src/net/pottercraft/ollivanders2/Ollivanders2.java

```diff
@@ -2062,6 +2062,17 @@ public void spellCoolDownMessage(@NotNull Player player)
       player.sendMessage(chatColor + "You are too tired to cast this spell right now.");
    }
 
+   /**
```

Comment:
You don't need to put this here - you should keep the logic for this inside the APPARATE spell class. It should look exactly like line 120 in APPARATE.java.